tests/end2end/MeetingSeries_security.js

```diff
@@ -0,0 +1,100 @@
+import { E2EGlobal } from './helpers/E2EGlobal'
+import { E2EApp } from './helpers/E2EApp'
+import { E2ESecurity } from './helpers/E2ESecurity'
+import { E2EMeetingSeries } from './helpers/E2EMeetingSeries'
+
+describe('MeetingSeries Security', function () {
+    const insertMeetingSeriesMethod = "meetingseries.insert";
+    const removeMeetingSeriesMethod = "workflow.removeMeetingSeries";
+    const updateMeetingSeriesMethod ="meetingseries.update";
+
+    beforeEach("goto start page and make sure test user is logged out", function () {
+        E2EApp.gotoStartPage();
+        E2EApp.loginUser();
+        expect(E2EApp.isLoggedIn()).to.be.true;
+    });
+
+    before("reload page and reset app", function () {
+        E2EApp.resetMyApp(true);
+        E2EApp.launchApp();
+    });
+
+    it('can not insert a new MeetingSerie if not logged in', function () {
+        let aProjectName = "Hacker Project #1";
+        let aMeetingName = "Hacker Meeting #1";
+        E2EApp.logoutUser();
+        expect(E2EApp.isLoggedIn()).to.be.false;
+
+        E2ESecurity.expectMethodToExist(insertMeetingSeriesMethod);
+        E2ESecurity.replaceMethodOnClientSide(insertMeetingSeriesMethod);
+        
+        let noOfMeetingSeries = server.call('e2e.countMeetingSeriesInMongDB');
+
+        E2ESecurity.executeMethod(insertMeetingSeriesMethod, {project: aProjectName, name: aMeetingName});
+        expect(server.call('e2e.countMeetingSeriesInMongDB')).to.equal(noOfMeetingSeries);
```

Comment:
It would be nice if you could add a description to each of the following `expect`-statements. Since there are more than one `expect`-statements in this test case it is easier to identify which one made the test red if the have unique descriptions.

E.g. `expect(server.call('e2e.countMeetingSeriesInMongDB'), 'Meeting Series should not be added if user is not logged in').to.equal(noOfMeetingSeries);`